docs/source/en/main_classes/pipelines.mdx

```diff
@@ -314,6 +314,11 @@ Pipelines available for audio tasks include the following.
     - __call__
     - all
 
+### ZeroShotAudioClassificationPipeline
+
+[[autodoc]] ZeroShotAudioClassificationPipeline
+    - __call__
+    - all
```

Comment:
Let's add the pipeline in a followup PR. Adding a model is already big enough for one PR.